apiserver/common/errors.go

```diff
@@ -100,6 +100,9 @@ type RedirectError struct {
 	// CACert holds the certificate of the remote server.
 	CACert string `json:"ca-cert"`
 
+	// ControllerTag uniquely identifies the controller being redirected to.
+	ControllerTag names.ControllerTag
```

Comment:
This is missing the json tag for the controller tag.

Reply:
Sorted.